README.md

```diff
@@ -16,13 +16,21 @@ go build -o proxmox-api-go
 
 ## Run
 
+Create a local `.env` file in the root directory of the project and add the following environment variables:
```

Comment:
@JamborJan I would leave the user the choice to keep with the classic `export` way along with .env loading

Reply:
Thanks for pointing that out @mleone87. It was not my intention to remove that functionality but I tested wrong and didn't catch that one. I fixed it now and some manually testing was successful. The projects test scripts are not able yet to include these aspects of testing. Maybe that's something for the future and we should open an issue for that. Not sure if #210 is the right place for that.

Let me know if you have any further feedback. Thanks again for considering my PR.